tests/qos/test_qos_sai.py

```diff
@@ -1616,6 +1616,9 @@ def testQosSaiDscpToPgMapping(
             "src_port_ip": dutConfig["testPorts"]["src_port_ip"],
         })
 
+        src_port_name = dutConfig["dutInterfaces"][testParams["src_port_id"]]
+        testParams['dscp_to_pg_map'] = load_dscp_to_pg_map(duthost, src_port_name, dut_qos_maps)
```

Comment:
@rbpittman  can we make it conditional for Cisco only, since broadcom-dnx run this case

Reply:
sure thing, added